coursebuilder/modules/explorer/_static/js/course.js

```diff
@@ -16,7 +16,11 @@ $(document).ready(function documentReady() {
 
   $('.course-carousel').on('changed.owl.carousel', function(event) {
     var pos = $(this).position();
-    window.scrollTo(pos.top, pos.left);
+    window.scrollTo({
```

Comment:
@Sharonsyra This fixed it?

Reply:
Yes

Reply:
@Sharonsyra why?

Reply:
I googled and the theory was that it was a browser issue but looks like it doesn't work on other browsers. It, however, works on chrome and safari though. Still working on this.

Reply:
Hey Chege, check the event.target version here https://sharon-dot-cfafrica-mooc-dev.appspot.com/test/unit?unit=4&lesson=13